primo/data_parser/metric_data.py

```diff
@@ -0,0 +1,602 @@
+#################################################################################
+# PRIMO - The P&A Project Optimizer was produced under the Methane Emissions
+# Reduction Program (MERP) and National Energy Technology Laboratory's (NETL)
+# National Emissions Reduction Initiative (NEMRI).
+#
+# NOTICE. This Software was developed under funding from the U.S. Government
+# and the U.S. Government consequently retains certain rights. As such, the
+# U.S. Government has been granted for itself and others acting on its behalf
+# a paid-up, nonexclusive, irrevocable, worldwide license in the Software to
+# reproduce, distribute copies to the public, prepare derivative works, and
+# perform publicly and display publicly, and to permit others to do so.
+#################################################################################
+
+# Standard libs
+import logging
+import pandas as pd
+
+# User-defined libs
+from primo.utils.config_utils import UserPriorities
+from primo.data_parser.default_data import (
+    SUPP_IMPACT_METRICS,
+    SUPP_EFF_METRICS,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable = protected-access
+# pylint: disable = too-many-arguments
+# pylint: disable = too-many-instance-attributes
+class Metric:
```

Comment:
Should we include the API documentation for these in readthedocs? If yes, could you please update the docs section? (See: https://github.com/NEMRI-org/primo-optimizer/tree/main/docs/Utilities)

Reply:
Per @radhakrishnatg , to be included in a future PR